test/Dockerfile

```diff
@@ -1,18 +1,26 @@
 FROM node:18.17.1-bullseye
 
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
```

Comment:
The default installation of Chromium in the test-runner image is no longer needed, and causes issues on MacOS.